syntax = "proto3";

// The information service definition.
service Informant {
  // Returns the information of the user.
  rpc getInfo (TokenRequest) returns (Information) {}
}

// The request message containing the token to verify the user's identity.
message TokenRequest {
  string token = 1;
}

// The response message containing the user's information.
message Information {
  string username = 1;
  string name = 2;
  string surname = 3;
  string email = 4;
}
